### DISCUSSION TITLE
syslog question

### DISCUSSION THREAD

**Discussion by ktroberson** [Security-Onion-Solutions/securityonion#5517] [Unsupported Versions]
I am forwarding syslog from an AlienVault OSSIM using rsyslog. I tcpdump the interface I am sending it to on the Sec Onion box and I see the logs. I have run so-allow for the sending machine, but I dont see the data in the Onion. Please help

**Comment by Git-Me-Some-Hub** [comment]
You might want to do a TCPdump command to confirm they are making it there. If you are using default 514 port and you did the so-allow it should come. Not sure it will parse it correctly, though. 

Look at the Firewall KB: 
- https://docs.securityonion.net/en/2.3/firewall.html

TCPDump Commands: 
- tcpdump src <ip>
- tcpdump -nnSX port <port>

Confirm IP is right in allow list: 
- /opt/so/saltstack/local/salt/firewall/hostgroups.local.yaml

**Comment by ben-sec** [comment]
I experienced the same issue and in my special case the source IP of my Syslog-Server fell accidentally into the same IP address space like the local SO docker containers. This caused the so called [Reverse Path Forwarding filtering](https://access.redhat.com/solutions/53031) in the OS to just drop the packets even though I saw them on the interface. Maybe you experience the same thing?

Where are you looking in SO? In Kibana, go to Discover and select the "*:so-*" Index and filter on "event.module: syslog". There you should find your (unparsed) syslog messages.

Cheers, Ben

**Comment by cyb3rz3us** [comment]
### Update-1

Just tried forwarding the logs via UDP and observe the same result: UDP port 514 is open, logs seen via tcpdump, no logs seen via SO console webpage.

### Original post

I have this exact same issue with one notable difference - I am forwarding the logs from my primary syslog server to the SO box.  The logs are coming over via TCP and I have confirmed the following-on the SO server:

- TCP port 514 is open
- The logs are being rcv'd as shown via: `sudo tcpdump -nnvv -i any dst port 514`
- The forwarding host is seen when executing: `sudo so-firewall includedhosts syslog`
- The forwarding host is in the following: `/opt/so/saltstack/local/salt/firewall/hostgroups.local.yaml`

Despite the above, when I check via the primary SO console webpage, I see no logs with a source IP of either the **(a)** forwarding server; or **(b)** the original sender.

Also, I do know that the logs will not parse properly but I view that as secondary because I need to get the data to SO before moving on to crafting regex, Grok, pipelines, etc.

Any insights, suggestions, etc. are welcome...

**Reply by dougburks** [reply]
@cyb3rz3us From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.

**Reply by cyb3rz3us** [reply]
Thanks @dougburks - will do.  I misinterpreted that that readme section to mean truly different issues but given these are 99% the exact same symptoms, I mistakenly thought it made sense to post here.  Thanks for clarifying and won't happen again...

**Comment by gebhard73** [comment]
Hi,

is `so-status` reporting everything up, especially filebeat?

What do the filebeat stats say?
`curl -s localhost:5066/stats | jq`
